src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import React, { useState, ChangeEvent } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find((user) => user.id === userId);
+
+  return foundUser || null;
+}
+
+const startingArray: Todo[] = todosFromServer.map(todo => ({
```

Comment:
`startingArray` of what? preparedTodos or todosWithUser or something similar describe it better